src/riak_kv_reaper.erl

```diff
@@ -190,6 +262,64 @@ redo() -> true.
 
 -type preflist_entry() :: {non_neg_integer(), node()}.
 
+-spec setup_reap(
+    riak_object:bucket(), riak_object:key()) ->
+        {non_neg_integer(), riak_core_apl:preflist_ann()}.
+setup_reap(Bucket, Key) ->
+    TombPause = app_helper:get_env(riak_kv, tombstone_pause, ?TOMB_PAUSE),
+    BucketProps = riak_core_bucket:get_bucket(Bucket),
+    DocIdx = riak_core_util:chash_key({Bucket, Key}, BucketProps),
+    {n_val, N} = lists:keyfind(n_val, 1, BucketProps),
+    {
+        TombPause,
+        riak_core_apl:get_apl_ann(
+            DocIdx,
+            N,
+            riak_core_node_watcher:nodes(riak_kv)
+        )
+    }.
+
+%% @doc
+%% Redo by indicating the original request was not successful (false)
+-spec maybe_redo(boolean()) -> boolean().
+maybe_redo(Redo) ->
+    if Redo -> false; true -> true end.
```

Comment:
This is a boolean negation. Given the type `boolean() -> boolean()` you could as well write as body `not Redo`.
Which might be easier to read for some people.

Reply:
Updated the code, and the comments to make it clearer why negation is happening